python/plugins/processing/algs/qgis/SetRasterStyle.py

```diff
@@ -48,7 +48,7 @@ def __init__(self):
         super().__init__()
 
     def flags(self):
-        return super().flags() | QgsProcessingAlgorithm.FlagNoThreading
+        return super().flags() | QgsProcessingAlgorithm.FlagNoThreading | QgsProcessingAlgorithm.FlagHideFromToolbox
```

Comment:
The explicit "FlagDeprecated" would be more appropriate here (it also hides from the modeler, and would allow us to easily identify deprecated algorithms in future...)

Reply:
Done